packager/packager.h

```diff
@@ -133,6 +133,8 @@ struct StreamDescriptor {
   bool dash_only = false;
   /// Set to true to indicate that the stream is for hls only.
   bool hls_only = false;
+
+  std::string rep_id;
```

Comment:
in my experience allowing non-numeric representation IDs is bound to cause problems, last time I tried I believe ExoPlayer failed to parse non-numeric representation IDs for example.

Reply:
instead of having the user specify the representation ID, what about allowing to specify using the representation ID in the output filename and then substitute in the computed representation ID by packager.